packages/admin-ui/src/number-input/number-input.style.ts

```diff
@@ -0,0 +1,32 @@
+import { style } from '@vtex/admin-ui-core'
+
+export const container = style({
+  ':not(:hover) > button': {
+    display: 'none',
+  },
+})
+
+export const input = style({
+  '::-webkit-inner-spin-button ': {
```

Comment:
I'm wondering whether we should really go with this pseudoelement as [it is marked as non-standard and not recommended for production](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-inner-spin-button). I might be missing something!

Reply:
We have to hide the native spin button and replace it for our custom spin button. In the worst case, the native spin button will continue to appear. At the moment, this is the way of "accessing" the webkit spin button style to override it.